libs/positions/src/lib/positions-table.tsx

```diff
@@ -85,7 +85,20 @@ export const PositionsTable = forwardRef<AgGridReact, Props>(
         components={{ AmountCell, PriceFlashCell, ProgressBarCell }}
         {...props}
       >
-        <AgGridColumn headerName={t('Market')} field="marketName" />
+        <AgGridColumn
+          headerName={t('Market')}
+          field="marketName"
+          cellRenderer={({
+            value,
+            data,
+          }: VegaICellRendererParams<Position, 'marketName'>) =>
+            value ? (
+              <Link href={`/#/markets/${data?.marketId}`}>{value}</Link>
```

Comment:
suggestion (blocking): A few things to consider here

1. Hardcoding the '/#/' could be problematic. Other apps don't use hash routing (Of course the trading app is the only one using this component, but we should try to follow the principles of the monorepo). Normally you should use the Link component from react-router-dom and it will handle the hash for you.
2. @asiaznik  recently added some helpers for url construction it looks something like `to={Links[Routes.MARKET](market.id)}`

What I would say is given that this is in a lib. The positions table should take a new optional prop `onMarketClick` which you can use to pass a function `() => navigate(Links[routes.MARKET])(market.id)`.

Unfortunately this results in quite a bit of prop drilling, and might not be the best solution long term but for now it will be more robust

Reply:
Also to ensure that we get consistent link styles, you can still use the ui-toolkit `Link` component with an onClick handler, just omit the href prop and it will render a span with the same styles

Reply:
ok, will do these changes to Orders and Fills tabs too

Reply:
updated. please review!